roles/fairfood/templates/post-receive.j2

```diff
@@ -25,7 +25,10 @@ upgrade_application() {
   ./script/install-bundler
 
   # Install the required gems
-  bundle --path "$HOME/.gem" --deployment --without "development" "test" | grep -v '^Using'
+  bundle config set --local deployment "true"
+  bundle config set --local path "$HOME/.gem"
+  bundle config set --local without "development test"
```

Comment:
My first thought was "if this is config, why don't we set it just the first time, rather than every time?" 
But I started thinking about how to, and it's not really worth it. Let's move on!

Reply:
Yes, I had the same thought but where does the setup happen? Maybe when setting up rbenv in the first place? I just checked and the rbenv role installs rbenv for the user and doesn't know anything about the app. This config is specific to the app and the Rails environment.

We could put it in the install-bundler script in the app but it will then rely on on RAILS_ENV being set. So yes, let's move on.